kb/plane_geometry/formal_statements/statements/statements_scs/stat_isosceles_triangle_circle_side_of_angle_radius_segment.scs

```diff
@@ -0,0 +1,14 @@
+stat_isosceles_triangle_circle_side_of_angle_radius_segment <- statement;;
+
+stat_isosceles_triangle_circle_side_of_angle_radius_segment -> rrel_key_sc_element: concept_isosceles_triangle; concept_circle; nrel_side; nrel_radius; concept_angle; concept_segment;;
+
+stat_isosceles_triangle_circle_side_of_angle_radius_segment => nrel_main_idtf: [Акс.(равнобедренный треугольник;окружность;радиус*;сторона*;угол;отрезок)] (*<- lang_ru;;*);;

```

Comment:
Absolute concepts should be mentioned before relations if there is no equal identifiers